snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java

```diff
@@ -58,8 +59,7 @@ public static SnomedOWLRelationshipDocument createFrom(final SnomedRelationship
 
 	private final String typeId;
 	private final String destinationId;
-	private final Integer integerValue;
-	private final Double decimalValue;
+	private final BigDecimal numericValue;
```

Comment:
What about backward compatibility? This will break any existing deployment where the two removed fields were present. Could you please add them back as read-only fields? Thanks!